portal/index.md

```diff
@@ -150,13 +150,13 @@ the example data used by the [Pythia Foundations Book](https://foundations.proje
 ## Join us!
 
 If you have questions or want to share anything with the
-[Project Pythia Team](#the-project-pythia-team), please reach out to us on our
-[GitHub Discussions page](https://github.com/ProjectPythia/projectpythia.github.io/discussions)
+[Project Pythia Team](#the-project-pythia-team), please reach out to us through the
+[Project Pythia category on the Pangeo Discourse forum](https://discourse.pangeo.io/c/education/project-pythia/)
 or join us at our Weekly Working Group Meetings.
```

Comment:
I was going to suggest linking directly to the calendar section here, like:
```suggestion
or join us at our weekly [Working Group Meetings](https://projectpythia.org/#meeting-event-calendar).
```
But then I just discovered the calendar page is broken, https://github.com/ProjectPythia/projectpythia.github.io/issues/290